src/Analyzers/Analyzers.csproj

```diff
@@ -12,12 +12,12 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.1</VersionPrefix>
-    <VersionSuffix>preview.1</VersionSuffix>
+    <VersionSuffix>preview</VersionSuffix>
```

Comment:
Are we releasing this as a preview package for now or should I remove `preview`?

Reply:
if we say GA this analyzer, I think we should remove the preview.

Reply:
@lilyjma - are we releasing the Analyzer as preview or GA?

Reply:
I updated the version to 0.1.0.